internal/amt/commands.go

```diff
@@ -37,12 +37,13 @@ type CodeVersions struct {
 
 // InterfaceSettings ...
 type InterfaceSettings struct {
-	IsEnabled   bool   `json:"isEnable"`
-	LinkStatus  string `json:"linkStatus"`
-	DHCPEnabled bool   `json:"dhcpEnabled"`
-	DHCPMode    string `json:"dhcpMode"`
-	IPAddress   string `json:"ipAddress"` //net.IP
-	MACAddress  string `json:"macAddress"`
+	IsEnabled    bool   `json:"isEnable"`
+	LinkStatus   string `json:"linkStatus"`
+	DHCPEnabled  bool   `json:"dhcpEnabled"`
+	DHCPMode     string `json:"dhcpMode"`
+	AmtIPAddress string `json:"amtipAddress"` //net.IP
+	OsIPAdress   string `json:"osipAddress"`
```

Comment:
```suggestion
	OsIPAdress   string `json:"osIpAddress"`
```

use consistent naming style -- camelCase